Library/Dungeon Fantasy/Dungeon Fantasy 6/Dungeon Fantasy 6 Equipment.eqp

```diff
@@ -0,0 +1,234 @@
+{
+	"type": "equipment_list",
+	"version": 4,
+	"rows": [
+		{
+			"id": "6a106f3c-5bbc-4c5a-b7b4-9e0f913c370c",
+			"type": "equipment",
+			"description": "Bracers of Force",
+			"reference": "DF6:5",
+			"reference_highlight": "Bracers of Force",
+			"notes": "11 FP as power item",
```

Comment:
It looks like most of the Bracers of Force have the same FP rating as a power item... shouldn't that vary depending on value?

Reply:
I saw that too. The Forcefield DR is some kind of enchantment and as I understand it enchantments don't count towards power item FP, and the bracers have a significant amount of DR themselves, so perhaps the material value accounts for the 11 FP.

At least, that's how I'm interpreting the section describing different forcefield DR values at different prices but only one listing for its worth as a power item.